grpc/server.proto

```diff
@@ -37,8 +37,23 @@ message AckRequest {
 
 message AckResponse {}
 
+message DeleteTopicRequest {
+    string topic = 1;
+}
+
+message DeleteTopicResponse {}
+
+message DeleteSubscriptionRequest {
+   string topic = 1;
+   string subscription_id = 2;
+}
+
+message DeleteSubscriptionResponse {}
```

Comment:
On return of response we aren't expecting anything?

Reply:
It a common practice to return nothing on Delete, amazon doing the same thing